### DISCUSSION TITLE
Mavis 4 tac_plus

### DISCUSSION THREAD

**Discussion by pavelteplitskiy** [MarcJHuber/event-driven-servers#36] [Q&A] [answered]
Hello.
Do you have a guide how to install mavis extention on tacacs plus server (running on centos)?

**Comment by MarcJHuber** [comment, marked as answer]
Hi,

no, sorry, I don't have any CentOS (or other distribution specific) instructions. For a install with basic feature set you'll need perl, GNU make, a C compiler (preferably clang), the PCRE2 development libraries/headers plus the other stuff mentioned in PREREQUISITES.txt. A simple "./configure --minimum tac_plus-ng" (or ... tac_plus) followed by "make" and possibly "sudo make install" should then result in working setup.

Cheers,

Marc
